plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -49,6 +49,59 @@ type Props = {
   fields?: FormProps<any>['fields'];
 };
 
+export function getUiSchemasFromSteps(steps: Step[]): UiSchema[] {
+  const uiSchemas: Array<UiSchema> = [];
+  steps.forEach(step => {
+    const schemaProps = step.schema.properties as JsonObject;
+    for (const key in schemaProps) {
+      if (schemaProps.hasOwnProperty.call(schemaProps, key)) {
```

Comment:
Not sure that we need the .call here either?

`schemaProps.hasOwnProperty(key)` should be fine?

Reply:
Totally agree... Fixed!